tests/test_transforms/test_transform_wrappers.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import warnings
```

Comment:
This file should be removed since it's duplicated with `tests/test_transform/test_transform_wrappers.py`